### PR TITLE
.item() method promoted to Stage 3, renamed to .at(), implemented by Firefox 85

### PR DESCRIPTION
**Pull request by afmenez**
Ditto.

### CODE REVIEW

**Review by ljharb — CHANGES_REQUESTED** [review]
We should hold off on landing this due to https://github.com/tc39/proposal-item-method/issues/31 and https://github.com/tc39/proposal-item-method/issues/28

**Comment by afmenez** [comment]
Is the proposal moving back to Stage 2?

**Comment by ljharb** [comment]
It’s unclear; we’ll surely discuss it at the November meeting.

**Comment by afmenez** [comment]
@ljharb, is it OK now?

**Review by ljharb — APPROVED** [review]

**Review by zloirock — CHANGES_REQUESTED** [review]
https://github.com/zloirock/core-js#relative-indexing-method

**Comment by ljharb** [comment]
I don’t understand the reason for the block, can you elaborate?

**Comment by zloirock** [comment]
`core-js` implementation was not added. Any reason to do it in another PR?

**Comment by ljharb** [comment]
ah, thanks. while i don't think we should be blocking proposal changes based on adding test results (as long as the result won't be false positives in the interim), it seems fine to add core-js here.

**Comment by afmenez** [comment]
I am not familiar with corejs, and I don't know which versions are and are not compatible.  Do you mind adding it in a separate patch?

**Review by chicoxyzzy — APPROVED** [review]

**Comment by zloirock** [comment]
@afmenez just add
```js
babel7corejs3: babel.corejs,
typescript4corejs3: typescript.corejs,
```
to `Array#at` and `%TypedArray%#at` results.

**Comment by zloirock** [comment]
Oops

File: data-esnext.js [hunk 1]
```diff
@@ -1680,17 +1688,20 @@ exports.tests = [
              return true;
            }
            var arr = new Constructor([1, 2, 3]);
```
Comment (zloirock) [inline data-esnext.js:1690]: `new BigInt64Array([1, 2, 3]) -> TypeError`

File: data-esnext.js [hunk 2]
```diff
@@ -1680,17 +1688,20 @@ exports.tests = [
              return true;
```
Comment (chicoxyzzy) [inline data-esnext.js]: I think it should be `false` here

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
there are a few issues in %TypedArray% test which need to be addressed

File: data-esnext.js (hunk 1 above)
Comment (afmenez) [reply data-esnext.js:1690]: Fixed

Comment (ljharb) [reply data-esnext.js:1690]: Let's fix this by adding a separate BigInt test rather than just not testing them at all?

**Review by zloirock — APPROVED** [review]

File: data-esnext.js (hunk 2 above)
Comment (afmenez) [reply data-esnext.js]: Fixed.

File: data-esnext.js (hunk 1 above)
Comment (afmenez) [reply data-esnext.js:1690]: Please feel free to add it in a separate patch.  This one is already 10 times the work I originally intended to do.

**Review by chicoxyzzy — APPROVED** [review]
Removing BigInt typed arrays tests is a downgrade, but given that old test wasn't working before, I think that we can add them later (but we still need to, because if I'm not mistaken, there is a WIP patch for JavaScriptCore and JSC doesn't support BigInt typed arrays yet)

**Pull request merged**